### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by zacharyestep**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -149,252 +189,294 @@ def save_results(analysis_results):
...
+        "SELECT md5hash FROM storefiles WHERE present_locally = TRUE AND timestamp >= '{0}' "
+        "ORDER BY timestamp DESC".format(start_date_binaries)
+    )
+
+    return cur
+
+
+def execute_script() -> None:
+    """
+    Execute a external maintenence script (vacuum script).
```
Comment (dseidel-b9) [inline main.py]: *maintenance

File: README.md [hunk 2]
```diff
@@ -204,6 +172,28 @@ The Yara agent must be installed on the same system as Cb Response.
 	```
 # Development Notes	
 
+## Vacuum Script
```
[the remark this answers is not in the document]
Comment (dseidel-b9) [reply README.md]: As @bchoinski-cb suggested, let's use "maintenance" in place of "vacuum"

File: binary_database.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
+# coding: utf-8
+# Copyright © 2014-2019 VMware, Inc. All Rights Reserved.
+
 import logging
 
 from peewee import *
-from playhouse.sqliteq import SqliteQueueDatabase
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
```
[the remark this answers is not in the document]
Comment (bchoinski-cb) [reply binary_database.py]: What hardcoded?

File: tasks.py [hunk 4]
```diff
@@ -1,53 +1,26 @@
...
-    if 'broker_url' in config['general']:
-        app.conf.update(
-            broker_url=config['general']['broker_url'],
-            result_backend=config['general']['broker_url'])
-
-    return True
-
 
 def add_worker_arguments(parser):
     parser.add_argument('--config-file', default='yara_worker.conf', help='Yara Worker Config')
```
[the remark this answers is not in the document]
Comment (bchoinski-cb) [reply tasks.py]: It defaults to a cwd file, if it does not exist an exception will be raised anyways.  The README instructions have the user creating such a file.

File: binary_database.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (dseidel-b9) [reply binary_database.py]: `logger.setLevel(logging.DEBUG)`

**Pull request closed without merging**